app/controllers/katello/api/v2/api_controller.rb

```diff
@@ -38,6 +36,7 @@ def resource_class
       @resource_class ||= "Katello::#{resource_name.classify}".constantize
     end
 
+    # rubocop:disable MethodLength
```

Comment:
@daviddavis 2 lines previously; 3 lines now... this must have been failing on master for some time now?